api.go

```diff
@@ -61,3 +62,18 @@ func (es *ExploreServer) apiGetBlockData(start types.BlockHeight, end types.Bloc
 
 	return blockSummaries, err
 }
+
+// apiGetHash queries the running instance of siad and returns the raw
+// data to the calling function. The json data could decode into
+// multiple types of structures, all of which *should* have the
+// ResponseType field as a string for identifying the type of data
```

Comment:
it depends on what you are explaining. types.go makes sense to have giant docstrings because the entire protocol is being defined.